libraries/src/AWS.Lambda.Powertools.Logging/Internal/PowertoolsConfigurations.cs

```diff
@@ -27,7 +27,7 @@ internal static class PowertoolsConfigurationsExtension
     /// <summary>
     ///     Gets the log level.
     /// </summary>
-    /// <param name="powertoolsConfigurations">The Powertools configurations.</param>
+    /// <param name="powertoolsConfigurations">The Powertools for AWS Lambda (.NET) configurations.</param>
```

Comment:
I wouldn't change here. comment in code

Reply:
We're unable to use "Powertools" as a standalone brand any more due to internal issues